src/com/TigersIter2/areaEffects/AreaEffect.java

```diff
@@ -10,8 +14,19 @@
  */
 public abstract class AreaEffect {
 
-    private Location location;
-    public StatsModifier statsMod;
+    private Location location;  //This is the location used by MODELS to determine where the areaEffect is
+    private Location pixelLocation; //This is the location used by VIEWS to determine where the areaEffect is
```

Comment:
Shouldn't this be a concern for the views to figure out? Why does the model need to know about its visual representation, or even that pixels exist?